lib/classes/databox.php

```diff
@@ -16,6 +16,79 @@
 
 class databox extends base
 {
+
+    public static function getAll(Application $app, \appbox $appbox)
+    {
+        try {
+            $rows = $appbox->get_data_from_cache(static::CACHE_BASE_DATABOX . '_all');
```

Comment:
Be careful, no exception is thrown by memcache when value is not present. it sometimes only return `false` instead.